src/gpu/sycl/prelu_kernels.hpp

```diff
@@ -0,0 +1,562 @@
+/*******************************************************************************
+* Copyright 2022 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#ifndef GPU_SYCL_PRELU_KERNELS_HPP
+#define GPU_SYCL_PRELU_KERNELS_HPP
+
+#include <cmath>
+#include <cstdlib>
+#include <stdlib.h>
+#include "CL/sycl/builtins.hpp"
```

Comment:
What is this file used for? I don't see it in the specification. I think it's not intended for external use.

Reply:
Yes, this file is unused and hence removed.